frontend/src/constants/environments.ts

```diff
@@ -1,38 +1,45 @@
+import { stringToBoolean } from "src/utils/generalUtils";
+
 const {
   NODE_ENV,
   NEXT_PUBLIC_BASE_PATH,
-  USE_SEARCH_MOCK_DATA = "",
+  USE_SEARCH_MOCK_DATA,
   SENDY_API_URL,
   SENDY_API_KEY,
   SENDY_LIST_ID,
   API_URL,
-  API_AUTH_TOKEN = "",
+  API_AUTH_TOKEN,
+  NEXT_BUILD,
+  SESSION_SECRET,
   NEXT_PUBLIC_BASE_URL,
-  FEATURE_SEARCH_OFF = "false",
-  FEATURE_OPPORTUNITY_OFF = "false",
-  NEXT_BUILD = "false",
-  SESSION_SECRET = "",
+  FEATURE_SEARCH_OFF,
+  FEATURE_OPPORTUNITY_OFF,
+  FEATURE_AUTH_OFF,
```

Comment:
This feels cleaner to grab them here and set defaults in the export. Would like to alphabetize them as well, might do that in a follow-up PR.

Reply:
agree alphabetizing is a good idea, also agree it might be cleaner to do it later on